ui/app/templates/components/pricing-metrics-config.hbs

```diff
@@ -1,13 +1,47 @@
-{{#if (eq mode "edit")}}
+{{#if (eq @mode "edit")}}
   <form onsubmit={{action "onSaveChanges"}} data-test-pricing-metrics-config-form>
     <div class="box is-sideless is-fullwidth is-marginless">
-      <MessageError @model={{model}} @errorMessage={{error}} />
-      {{#each model.configAttrs as |attr|}}
-        <FormField
-          data-test-field
-          @attr={{attr}}
-          @model={{model}}
-        />
+      <MessageError @model={{@model}} @errorMessage={{this.error}} />
+      {{#each @model.configAttrs as |attr|}}
+        {{#if (and (eq attr.type "string") (eq attr.options.editType "boolean"))}}
+          <label class="is-label">Usage data collection</label>
+          {{#if attr.options.helpText}}
+              <p class="sub-text">{{attr.options.helpText}} {{#if attr.options.docLink}}<a href="{{attr.options.docLink}}" target="_blank" rel="noopener noreferrer">See our documentation</a> for help.{{/if}}</p>
+          {{/if}}
+          <div class="control is-flex has-bottom-margin-l">
+            <input data-test-field type="checkbox" id="{{attr.name}}" name={{attr.name}} class="switch is-rounded is-success is-small" checked={{eq (get @model attr.name) attr.options.trueValue}} 
+            onchange={{action (action "updateBooleanValue" attr) value="target.checked"}} />
+            <label for="{{attr.name}}">
+              {{#if (eq @model.enabled "Off")}}
+                Data collection is off
+              {{else}}
+                Data collection is on
+              {{/if}}
+            </label>
+          </div>
+          {{else if (eq attr.type "number")}}
+            <div class="has-top-margin-s">
+              <label for="{{attr.name}}" class="is-label">
+                {{ attr.options.label }}
+              </label>
+              {{#if attr.options.subText}}
+                <p class="sub-text">{{attr.options.subText}} {{#if attr.options.docLink}}<a href="{{attr.options.docLink}}" target="_blank" rel="noopener noreferrer">See our documentation</a> for help.{{/if}}</p>
+              {{/if}}
+              <div class="control">
+                <input 
+                  data-test-field
+                  id={{attr.name}}
```

Comment:
no chance there would be two of the same fields on the same page?  e.g. two with the same attr.name?

Reply:
Nope, should not happen.